### DISCUSSION TITLE
Mesh in the downsteam region of a fan

### DISCUSSION THREAD

**Discussion by Joshuaklein98** [firemodels/fds#13875] [Q&A]
Hello,

I want to model a jet fan in a tunnel and I would like to know how to define the mesh in the downstream region of a fan.

Let's suppose that my fan has a HVAC inlet vent and a HVAC outlet vent with dimensions of 0,4m x 0,4m. Is it OK to use a mesh with cells of 0,4m x 0,4m x 0,4m in the downstream region of the fan in the tunnel, or should I refine my mesh? In that case, how long in the tunnel do I have to have this mesh?

I have not seen this stuff detailed in the User's Guide.

Thank you very much.

**Comment by drjfloyd** [comment]
What resolution you need to use is not something we can tell you. It depends on what exactly you are doing and how good of an answer you need. 

Do you need to know the near field of the jet fan? If that is the case then 1 cell across the fan isn't going to resolve the near field. 
Do you just care about what is happening some long distance away from the fans where you just care about the bulk flow? Then maybe 1 cell is adequate.

You may wish to do a grid study to see how the grid changes the outputs that you ultimately care about.

**Comment by Joshuaklein98** [comment]
Hello, Mr Floyd, thanks for your answer.

In my case, it is more important what happens far from the fan.

I will try a grid study, as you suggest.

Cheers.

**Comment by Er9y714** [comment]
In case you are not modelling the same fan outlet area in FDS as the fan specs, I would suggest not only looking into volume flow rate of the fan but also the thrust value.
